index.html

```diff
@@ -1,54 +1,273 @@
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/hability.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Multi-Post Stories</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads; no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button multi">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/Snapshoot Portfolio.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/profesional.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Multi-Post Stories</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button multi">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
```

Comment:
- [ ] **[OPTIONAL]** Kindly, Make sure all buttons have button interaction like hover, enabled, pressed. You can do that by using the pseudo class `:hover` for the `hover` interaction, `:active` for the `enabled` interaction, `:focus` for the `pressed` interaction. 

This will make you project more interactive and user friendly 📱😄

Reply:
@Trast00 didn't have time to do the optional!! thanks for the review!